cypress/integration/learn/common-components/searchBar.js

```diff
@@ -27,6 +27,38 @@ describe('Search bar', () => {
     cy.get('.ais-SearchBox').should('be.visible');
   });
 
+  it('Should not display hits', () => {
+    search('freeCodeCamp');
+    cy.get('.ais-Hits-list').should('not.exist');
```

Comment:
[Ideally](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) we shouldn't rely on implementation details (like classes) for tests.  Could you change this to use a `data-cy` attribute?  E.g. 
```suggestion
    cy.get('.[data-cy=hits-list]').should('not.exist');
```

This applies more generally to the tests in this file, though if those changes would be better in another PR, that's fine.

Reply:
Some of the classes that we are targeting are coming straight from the algolia module, but I will add data-cy to what I can.

Reply:
Yeah, I understand.  I think this is slightly safer, though, since no-one is going to add `data-cy` for any reason other than as a target for testing.  Anyways, thanks for implementing this.